R/sort_levels.R

```diff
@@ -0,0 +1,67 @@
+#' Sort factor levels according to a user-defined function
+#'
+#' @param .f A factor (or character vector).
+#' @param .fun A function that will sort or permute the existing factor levels.
```

Comment:
```suggestion
#' @param .fun A function that will permute the existing factor levels.
```

Reply:
@hadley  : Given the emphasis on permutation in your edited version of the docs, would `fct_permute_levels` be a more appropriate name for the function?